### PR TITLE
Update start-installation.md

### PR DESCRIPTION
**Pull request by jiuly256**
Propongo mejorar la redacción en español

| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #16605 propongo estos cambios

### CODE REVIEW

File: docs/guide-es/start-installation.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ El primer comando instala [composer asset plugin](https://github.com/francoisplu
 que permite administrar dependencias de paquetes bower y npm a través de Composer. Sólo necesitas ejecutar este comando
 una vez. El segundo comando instala Yii en un directorio llamado `basic`. Puedes elegir un nombre de directorio diferente si así lo deseas.
 
-> Note: Durante la instalación, Composer puede preguntar por tus credenciales de acceso de Github. Esto es normal ya que Composer 
+> Nota: Durante la instalación, Composer puede preguntar por tus credenciales de acceso de Github. Esto es normal ya que Composer
```
Comment (samdark) [inline docs/guide-es/start-installation.md]: These should not be translated.

**Review by samdark — CHANGES_REQUESTED** [review]
Please revert changes to note blocks.

**Review by jiuly256 — COMMENTED** [review]
nota to note again

File: docs/guide-es/start-installation.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ y el correspondiente driver de base de datos (como `pdo_mysql` para bases de dat
 Configurar Servidores Web <span id="configuring-web-servers"></span>
 -------------------------
 
-> Info: Puedes saltear esta sección por ahora si sólo estás probando Yii sin intención
+> Información: Puedes saltear esta sección por ahora si sólo estás probando Yii sin intención
```
Comment (samdark) [inline docs/guide-es/start-installation.md]: Same here. Should be `Info` as it was.

**Review by jiuly256 — COMMENTED** [review]
Instalando only

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**